### PR TITLE
skeptipay-ui

### PR DESCRIPTION
**Pull request by KwaneleNdenze**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -3,11 +3,18 @@ class ApplicationController < ActionController::Base
 
   def layout_by_resource
     # user_signed_in? ? nil : "application"
-    is_a?(Devise::SessionsController) | is_a?(Devise::RegistrationsController) | is_a?(Devise::PasswordsController) ? "devise" : "application"
-   
+    if user_signed_in? 
+      "application_ui"
+    else
+      is_a?(Devise::SessionsController) | is_a?(Devise::RegistrationsController) | is_a?(Devise::PasswordsController) ? "devise" : "application"
```
Comment (thatosmk) [inline app/controllers/application_controller.rb:9]:
```suggestion
      devise_controller? ? "devise" : "application"
```

Comment (thatosmk) [reply app/controllers/application_controller.rb:9]: check if this works as you wish

File: app/controllers/profile_controller.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class ProfileController < ApplicationController
```
Comment (thatosmk) [inline app/controllers/profile_controller.rb:1]:
```suggestion
class ProfilesController < ApplicationController
```

File: config/routes.rb [hunk 3]
```diff
@@ -1,10 +1,20 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: {sessions: 'users/sessions', registrations: 'users/registrations', passwords: 'users/passwords'}
+
+  devise_scope :user do
+    get '/users/sign_out', to: 'devise/sessions#destroy'
+  end
```
Comment (thatosmk) [inline config/routes.rb:6]: this is not a fix or your issue, I know what you are struggling with. Try to debug and fix this without adding this

File: app/helpers/nav_active_link_helper.rb [hunk 4]
```diff
@@ -2,4 +2,8 @@ module NavActiveLinkHelper
   def active_link_css(path)
     current_page?(path) ? 'active' : ''
   end
+
+  def app_active_link_css(path)
+    current_page?(path) ? 'bg-cardBody text-gray-300 border border-gray-300 hover:text-white group flex items-center px-2 py-4 text-sm font-thin rounded-md' : 'text-gray-300 hover:bg-lightGreen hover:text-white group flex items-center px-2 py-4 text-sm font-thin rounded-md'
```
Comment (thatosmk) [inline app/helpers/nav_active_link_helper.rb:7]:
```suggestion
    current_page?(path) ? 'link link--active' : 'link'
```

Comment (thatosmk) [reply app/helpers/nav_active_link_helper.rb:7]: then in your tailwind css file
```css
.link {
  @apply text-gray-300 hover:bg-lightGreen hover:text-white group flex items-center px-2 py-4 text-sm font-thin rounded-md
}
.link--active {
  @apply what-ever-defines-active-state
}
```

Look at https://tailwindcss.com/docs/adding-custom-styles#adding-base-styles

**Review by thatosmk — CHANGES_REQUESTED** [review]
just minor changes but looks good so far

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (KwaneleNdenze) [reply app/controllers/application_controller.rb:9]: Made the change, works.

**Pull request merged**